Date/Human.php

```diff
@@ -103,7 +103,7 @@ class Date_Human
      * @access   public
      * @static
      */
-    function gregorianToHuman($day = 0, $month = 0, $year = 0)
+    static function gregorianToHuman($day = 0, $month = 0, $year = 0)
```

Comment:
It's the same problem and the solution I mention in the previous comment thread.